tests/performance/density.go

```diff
@@ -168,11 +167,9 @@ func createBatchVMIWithRateControl(virtClient kubecli.KubevirtClient, vmCount in
 }
 
 func createVMISpecWithResources() *kvv1.VirtualMachineInstance {
-	vmImage := cd.ContainerDiskFor("cirros")
 	const cpuLimit = "100m"
 	const memLimit = "90Mi"
-	cloudInitUserData := "#!/bin/bash\necho 'hello'\n"
-	vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(vmImage, cloudInitUserData)
+	vmi := libvmi.NewCirros()
```

Comment:
I am not sure this is really what we want. 
1. This doesn't use libvmi exclusively as we override memory below
2. We don't really follow the memory requirments here (128/256 Mi)
3.  We use different binding (not relevant to test as far as I can see)

Reply:
> 1. This doesn't use libvmi exclusively as we override memory below
I don't think this is a problem. There is no other way to set memory and cpu using libvmi.

> 2. We don't really follow the memory requirments here (128/256 Mi)
This point is irrelevant, because the memory is set below.

> 3. We use different binding (not relevant to test as far as I can see)
What binding do you mean? Can you explain?

Reply:
1. I think we would benefit from defining the option. (limits can be a local option)
2. You are right
3.
 ```
vmi := libvmi.NewCirros(
					libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
					libvmi.WithNetwork(v1.DefaultPodNetwork()),
				)
```

Reply:
done